### DISCUSSION TITLE
Anyone else experiencing issues only with 24h graphs?

### DISCUSSION THREAD

**Discussion by dstroot** [upptime/upptime#254] [Q&A] [answered]
<img width="574" alt="Screenshot 2021-02-11 092558" src="https://user-images.githubusercontent.com/1438457/107673854-34da4980-6c4b-11eb-8b51-0a1000fd5f14.png">

Everything else works just fine.  Can't figure it out...

Cheers!

**Comment by BBaoVanC** [comment]
How many times has the `Uptime CI` workflow run in the past 24 hours?

**Reply by dstroot** [reply]
Uptime CI or Graphs CI?

**Reply by dstroot** [reply]
Both appear to be running

**Reply by dstroot** [reply]
... running normally.

**Comment by AnandChowdhary** [comment, marked as answer]
This is because the response time collection only happens once per 24 hours, so the graph is expected to look like that. If you want it to track this more frequently, you can increase the frequency of the CI schedule: https://upptime.js.org/docs/configuration#ci-schedule

**Reply by BBaoVanC** [reply]
Wait... I have response time triggered once every 24 hours, but I still have multiple points on the 24h graph.
